### PR TITLE
4 view articles

### PR DESCRIPTION
**Pull request by simonhighfield**
I tried using a bootstrap grid (in a container, row, column structure), in order to help with the resizing of this page.

However, I couldn't get this to work. I'll come back to that later if there's time, as I want to move on to the functionality of other features now

### CODE REVIEW

File: src/components/Home.jsx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import ArticleList from "./ArticleList";
+
+export default function Home() {
+    return (
+        <>
+            <h1>Home</h1>
+            <ArticleList></ArticleList>
```
Comment (nc-rose-mullan) [inline src/components/Home.jsx:7]: This can be self-closing

File: src/components/ArticleList.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import ArticleCard from "./ArticleCard";
+import { useEffect, useState } from "react";
+import axios from "axios";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+import './ArticleList.css'
+
+const endpoint = "https://nc-news-fswv.onrender.com/api/articles"
```
Comment (nc-rose-mullan) [inline src/components/ArticleList.jsx:9]: Please consider separating out your api requests into their own file -  and also take a look at the axios.create method to make your code even DRYer!

File: src/components/ArticleCard.jsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+import Card from 'react-bootstrap/Card';
+import './ArticleCard.css'
+
+export default function ArticleCard(props) {
+    const { article } = props
+
```
Comment (nc-rose-mullan) [inline src/components/ArticleCard.jsx:6]: Please include a loading pattern to improve the user experience while they’re waiting for the page to render.

**Review by nc-rose-mullan — COMMENTED** [review]
A few comments to consider Simon, but overall this is doing what it needs to. I see what you mean about responsiveness, but I'm happy for that to wait until later in favour of functionality.

File: src/components/ArticleCard.jsx (hunk 3 above)
Comment (simonhighfield) [reply src/components/ArticleCard.jsx:6]: Done

File: src/components/ArticleList.jsx (hunk 2 above)
Comment (simonhighfield) [reply src/components/ArticleList.jsx:9]: Done

File: src/components/Home.jsx (hunk 1 above)
Comment (simonhighfield) [reply src/components/Home.jsx:7]: Done

**Pull request merged**